deployment/deployment.yml

```
@@ -152,6 +152,23 @@ spec:
             cpu: "10m"
           limits:
             cpu: "2000m"
+      - name: disk-usage
+        image: gcr.io/dd-decaf-cfbf6/metabolic-ninja:devel
+        command: ["python", "-m", "disk_usage.main"]
+        env:
+        - name: SENTRY_DSN
+          valueFrom:
+            secretKeyRef:
+              name: metabolic-ninja-staging
+              key: SENTRY_DSN
+        resources:
+          requests:
+            cpu: "10m"
+          limits:
+            cpu: "2000m"
```

It will probably never get there but that seems very high for the simple script.
```suggestion
            cpu: "1000m"
```

---

It might seem arbitrarily high for this service, but it is our general policy for all burstable services to be allowed to use up to 2 vCPUs at a time. As you say it's not relevant here, save for some unlikely bug.